common/app/model/content.scala

```diff
@@ -96,7 +96,7 @@ final case class Content(
   lazy val hasTonalHeaderByline: Boolean = {
     (cardStyle == Comment || cardStyle == Editorial || (cardStyle == SpecialReport && tags.isComment)) &&
       hasSingleContributor &&
-      metadata.contentType != DotcomContentType.ImageContent
+      !metadata.contentType.contains(DotcomContentType.ImageContent)
```

Comment:
So to be clear, this is the error right (along with the other similar cases)? I.e. we were checking inequality previously when we should have been checking not contains. And the lack of warning prevented us spotting these earlier.

Reply:
yep. we were comparing `metadata.contentType` which is an `Option` to a `DotcomContentType` and the compiler was totally fine with it 😩 

Funny thing: if I had written the equality test as `DotcomContentType.ImageContent != metadata.contentType` the compiler would have complained